src/core/Link/Link/Link.baseStyles.ts

```diff
@@ -2,6 +2,6 @@ import { SuomifiTheme } from '../../theme';
 import { css } from 'styled-components';
 import { baseStyles } from '../BaseLink/BaseLink.baseStyles';
 
-export const ExternalLinkStyles = (theme: SuomifiTheme) => css`
+export const LinkStyles = (theme: SuomifiTheme) => css`
   ${baseStyles(theme)}
```

Comment:
Using the same pattern from `StaticChip`. This file only passes forward the `BaseLink` styles because nothing else is needed. Another possibility would be to remove this file altogether and use `/BaseLink/BaseLink.baseStyles` directly in `Link.tsx`

Reply:
I think this approach is pretty clean 👍

Reply:
I'm okay with either way.

If following the existing, separate `*.baseStyles.ts` it might help as every component are (or should) be with same pattern.
But it is also little redundant to just keep the `Link.baseStyles.ts` hanging with no actual value given and just to be "for future" --> we can always add it back if we really need some customisation where the baseStyles is needed.

Reply:
No clear consensus against my choice, so I'll leave it :)